x-pack/plugins/ml/public/embeddables/single_metric_viewer/single_metric_viewer_embeddable.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Suspense } from 'react';
+import ReactDOM from 'react-dom';
+import { pick } from 'lodash';
+
+import { Embeddable } from '@kbn/embeddable-plugin/public';
+
+import { CoreStart } from '@kbn/core/public';
+import { i18n } from '@kbn/i18n';
+import { Subject } from 'rxjs';
+import { KibanaContextProvider, KibanaThemeProvider } from '@kbn/kibana-react-plugin/public';
+import { IContainer } from '@kbn/embeddable-plugin/public';
+import { DatePickerContextProvider, type DatePickerDependencies } from '@kbn/ml-date-picker';
+import { UI_SETTINGS } from '@kbn/data-plugin/common';
+import { EmbeddableSingleMetricViewerContainer } from './embeddable_single_metric_viewer_container_lazy';
+import type { JobId } from '../../../common/types/anomaly_detection_jobs';
+import type { MlDependencies } from '../../application/app';
+import {
+  ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE,
+  SingleMetricViewerEmbeddableInput,
+  AnomalyChartsEmbeddableOutput,
+  SingleMetricViewerServices,
+} from '..';
+import { EmbeddableLoading } from '../common/components/embeddable_loading_fallback';
+
+export const getDefaultSingleMetricViewerPanelTitle = (jobIds: JobId[]) =>
+  i18n.translate('xpack.ml.anomalyChartsEmbeddable.title', {
+    defaultMessage: 'ML single metric viewer chart for {jobIds}',
+    values: { jobIds: jobIds.join(', ') },
+  });
+
+export type ISingleMetricViewerEmbeddable = typeof SingleMetricViewerEmbeddable;
+
+export class SingleMetricViewerEmbeddable extends Embeddable<
```

Comment:
This looks like the actions menu from the anomaly charts embeddable. The 'Edit' action is the one for the charts, and needs to re-open the workflow for the SMV embeddable.

Would be nice to have an 'Open in Single metric viewer' action added.

<img width="230" alt="image" src="https://github.com/elastic/kibana/assets/7405507/ba5f05b5-1a50-4a21-b9d1-d70811bad220">

Reply:
Fixed the actions menu in a744275ea927041847ca79cb1c968bfa338bec04

Reply:
Added 'Open in single metric viewer' action as a follow up in the related issue.

Reply:
Just to confirm - will the `Edit` action be added in as a follow-up? Just checking that it isn't intended to be working in this PR.

Reply:
That's right - added to the follow-up. 👍